components/molecule/rating/src/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import Star from './components/Star'
+
+const SIZES = {
+  SMALL: 'small',
+  MEDIUM: 'medium',
+  LARGE: 'large'
+}
+
+const BASE_CLASS = `sui-MoleculeRating`
+const CLASS_CONTAINER_STARS = `${BASE_CLASS}-containerStars`
+const CLASS_LABEL = `${BASE_CLASS}-label`
+const CLASS_LINK = `${BASE_CLASS}--withLink`
+
+const MoleculeRating = ({
+  numStars,
+  value,
+  size,
+  label,
+  href,
+  target,
+  link,
+  linkFactory: Link
+}) => {
+  const className = cx(BASE_CLASS, `${BASE_CLASS}--${size}`, {
+    [CLASS_LINK]: link
+  })
+
+  const labelLink = link ? (
+    <Link
+      href={href}
+      target={target}
+      rel={target === '_blank' ? 'noopener' : undefined}
+    >
+      {label}
+    </Link>
+  ) : (
+    label
+  )
+
+  return (
+    <div className={className}>
+      <div className={CLASS_CONTAINER_STARS}>
+        {new Array(numStars)
+          .fill(0)
+          .map((_, index) => <Star key={index} index={index} value={value} />)}
+      </div>
+      <p className={CLASS_LABEL}>{labelLink}</p>
+    </div>
+  )
+}
+
+MoleculeRating.displayName = 'MoleculeRating'
+
+MoleculeRating.propTypes = {
+  /** Number of Stars */
+  numStars: PropTypes.number,
+
+  /** Value of Raitng */
+  value: PropTypes.number,
+
+  /** Label of Raitng */
+  label: PropTypes.string,
+
+  /** size */
+  size: PropTypes.oneOf(Object.values(SIZES)),
+
+  /** HTML element: if true, render a link. Otherwise render a button */
+  link: PropTypes.bool,
+
+  /** Target to be added on the HTML link */
+  target: PropTypes.string,
+
+  /** URL to be added on the HTML link */
+  href: PropTypes.string,
+
+  /** Factory used to create navigation links */
+  linkFactory: PropTypes.func
+}
+
+MoleculeRating.defaultProps = {
+  numStars: 5,
```

Comment:
Could you move this to default parameters instead defaultProps? As you have done in "components/Star.js"